addons/l10n_de/models/hr_timesheet.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models, fields, api, _
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    l10n_de_template_data = fields.Binary(compute='_compute_l10n_de_template_data')
+    l10n_de_document_title = fields.Char(compute='_compute_l10n_de_document_title')
+
+    def _compute_l10n_de_template_data(self):
+        for record in self:
+            record.l10n_de_template_data = []
+
+    def _compute_l10n_de_document_title(self):
+        for record in self:
+            record.l10n_de_document_title = 'Timesheet Entries'
```

Comment:
`account.analytic.line` are not "Timesheet Entries"?

Reply:
Effectively, this is the model defined in the url while looking at the timesheet entries. Furthermore with this new class, the bug has disappeared :)

Reply:
I see this
* [opw-2699563](https://www.odoo.com/web#id=2699563&cids=1&model=project.task&view_type=form) for `account_followup.report_followup_print_all`
* [opw-2659703](https://www.odoo.com/web#id=2659703&cids=1&model=project.task&view_type=form) for `account_sepa_direct_debit.sdd_mandate_form_report`

Where do "Timesheet Entries" or `account.analytic.line` come from? How to reproduce?

_______

Moreover, it is only valid to call an `account.analytic.line` a "Timesheet Entry" if it is linked to a `project_id`, which is not defined in the module `analytic` but in `hr_expense`

_______


They are defining `docs` in `hr_timesheet.report_timesheet`, so the fix in `l10n_de.external_layout_din5008` should do the trick. However, with the current fix, you have this, which is not optimal
![image](https://user-images.githubusercontent.com/44770049/145384821-f79f6307-aecd-4509-afe5-5687637f6f2a.png)

Reply:
Here is a link to the ticket reporting the bug with the timesheet: [opw-2649221](https://www.odoo.com/web#id=2649221&cids=1&menu_id=4720&action=333&active_id=49&model=project.task&view_type=form).

So the idea is to define a different title for all the classes that extends `account.analytic.line`. Is that correct ?

Reply:
Please add that ticket to the commit message then :)

And yes. But following the screenshot, it would be better to have an empty string than duplicating the title.